### DISCUSSION TITLE
documentation about pdf

### DISCUSSION THREAD

**Discussion by pubpub-zz** [py-pdf/pypdf#1702] [PDF Format Questions]
@petervwyatt
maybe you can help us.
with @MartinThoma, in order to improve pypdf, we have started to push some discussion about sponsoring as we are getting more issues reported by some companies. The main idea is to be able to get some books/documents such as the one accessible to the pdf association, to share within the core team. We may be able to be offered some books directly from some companies, but do you think it would be possible to paid an access for our team from a third company? We do not have for the moment any structure, will we need to build one ?

Thanks in advance for your patience and advice.

**Comment by petervwyatt** [comment]
There is an announcement coming in early April that will be of relevance to you... sorry but I cannot say anymore 😀.
Otherwise, if you can locate a sponsor for your membership, that would be fine. 

You might also be able to join ISO discussions via your national standards body to participate in ISO TC 171 SC 2 assuming your country is already a P- or O-member of ISO TC 171 SC 2 (see https://www.iso.org/committee/53674.html?view=participation). This will not give you access to PDF Association working or liaison group discussions, but it will give you full access to all past and present work in ISO TC 171 SC 2. Contact your national standards body to find out more.

Please also note the PDF Association's [Charter](https://www.pdfa.org/wp-content/uploads/2021/01/2021-PDFa-Charter.pdf) and [IPR policy](https://www.pdfa.org/intellectual-property-rights-ipr-policy/) which must be agreed to as they allow the Association to operate as an official ISO liaison as ISO have stringent rules. If you join ISO via your national standards body then you will also have to agree to something very similar.

Hope that helps.

**Comment by pubpub-zz** [comment]
@MartinThoma / @MasterOdin  and all a great annoucement has been re leased!

https://www.pdfa.org/announcing-no-cost-access-to-iso-32000-2-pdf-2-0/

thanks for your teasing @petervwyatt !

**Reply by MartinThoma** [reply]
That is awesome news :tada:
